### PR TITLE
Load model only once.

### PR DESCRIPTION
**Pull request by kacunningham413**
Load models only once. Note we need to load "gams" and "no gams" versions. This increases memory uses and required a larger container in Cloud Run. Requests are a bit faster (0-15%) with this.

### CODE REVIEW

File: api/app.py [hunk 1]
```diff
@@ -24,6 +23,8 @@
 basic_auth = BasicAuth(app)
 CORS(app, support_credentials=True)
 
+gams_model = Model(has_gams=True)
```
Comment (ivanmkc) [inline api/app.py]: Perfect, moved to global scope.

Comment (ivanmkc) [reply api/app.py]: However, if you look at https://github.com/thebaumlaboratory/PlasmoCount/blob/b3c776ead5f436fa6e3ed0aca7d8f1683f5f4e49/api/programs/model.py#L52

You'll see that the `gams` parameter is only used at prediction time. So you actually only need one Model (which internally has 3 `learner models`). Just pass in the `gams` parameter when you call `model.predict`.

```
                    if self.has_gams:
                        gam_pred = self.gam_model.predict(bbox_img)
                        if str(gam_pred[0]) == 'asexual':
                            ls_pred = self.ls_model.predict(bbox_img)
                        else:
                            ls_pred = [FloatItem(-1)]
```

Now you can have your performance cake and eat it too.

**Review by ivanmkc — CHANGES_REQUESTED** [review]
See comments about gams

**Comment by ivanmkc** [comment]
Maybe you can experiment with a container that consumes less memory after the change.

File: api/app.py (hunk 1 above)
Comment (kacunningham413) [reply api/app.py]: Updated predict method signature to take has_gams as an arg and now we only need to use one model (good catch). Confirmed that we can use the smaller container size that we were using previously.

**Review by ivanmkc — APPROVED** [review]
Looks great!

**Pull request merged**